packages/perseus/src/widgets/python-program.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * This widget is for embedding Khan Academy Python programs.
+ */
+
+import {StyleSheet, css} from "aphrodite";
+import PropTypes from "prop-types";
```

Comment:
Same comment as the editor. Please define a `Typescript` Props type instead of using `prop-types`.